src/ui/ListItem.tsx

```diff
@@ -0,0 +1,45 @@
+/** @license
+ *  Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy
+ *  of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+
+import * as React from "react";
+
+import { CSS } from "../lib/Constants";
+
+/**
+ * Interface for the properties assigned to the ListItem component.
+ * @interface
+ */
+interface IListItemProps {
+  title: string;
+  subtitle?: string;
+  inlineControl: boolean;
+  controlClass?: string;
+  control: any;
+}
+
+/** Returns a stateless page layout template. */
+export function ListItem(props: IListItemProps) {
+  let inline = props.inlineControl ? "inline" : "";
+  return (
+    <div className={`${CSS.RMX_LIST_ITEM} ${props.controlClass} ${inline}`}>
```

Comment:
Your spacing within braces doesn't seem consistent: some pairs have them and some don't.  I usually include it, but consistency is the most important.